### PR TITLE
Modulemeta demo design guidance

### PR DESCRIPTION
**Pull request by onishiweb**
Add a new property to the module meta endpoint to return the design guidelines markdown file.

Required for the Registry's new demos construction.

### CODE REVIEW

File: lib/modulemetadata.js [hunk 1]
```diff
@@ -12,6 +12,21 @@ function ModuleMetadata(options) {
 }
 
 ModuleMetadata.prototype = {
+
+	_getFileFromList: Q.async(function*(filenames, module, installation) {
```
Comment (JakeChampion) [inline lib/modulemetadata.js]: Could we have tests for this new method?

Comment (onishiweb) [inline lib/modulemetadata.js]: Sure. However I have no idea where to start doing them. So some help would be appreciated.

Comment (JakeChampion) [inline lib/modulemetadata.js]: I believe this is the file you should add the tests to -- https://github.com/Financial-Times/origami-build-service/blob/284087007313a3decf2b45711d59a846141416b1/test/metadataapitest.js

**Comment by onishiweb** [comment]
@JakeChampion @rowanmanning are we leaving the testing for now then?

**Comment by JakeChampion** [comment]
Yep, the tests for this application need to have a look through as a whole.

**Pull request merged**